<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDDownloadDialog</class>
 <widget class="QDialog" name="CMDDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>239</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>配方下载</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>39</y>
     <width>430</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCompleted">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QListView" name="listViewTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>200</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
